docs/CHANGES.md

```diff
@@ -61,6 +61,7 @@
   an instant room is created. [jcbrand]
 - Ensure consistent behavior from `show_controlbox_by_default` [jcbrand]
 - #365 Show join/leave messages for chat rooms.
+- #585 Fixes the duplication bug due to case sensivity in adding contacts.
```

Comment:
I did that because I thought the logs below that belongs to **New configuration setting:** section. Is that the case or should I add log at the end of this or somewhere else?